webviz_subsurface/plugins/_group_tree/_plugin.py

```diff
@@ -12,9 +11,20 @@
     Frequency,
 )
 
-from ._callbacks import plugin_callbacks
-from ._ensemble_group_tree_data import EnsembleGroupTreeData
-from ._layout import LayoutElements, main_layout
+from ._utils import EnsembleGroupTreeData
+from ._views import (
+    GroupTreeView,
+    GroupTreeViewElement,
+    ViewControls,
+    ViewFilters,
+    ViewOptions,
+)
+
+
+class PluginIds:
+    # pylint: disable=too-few-public-methods
+    class Views(str, Enum):
+        GROUPTREE_VIEW = "group-tree-view"
```

Comment:
You could follow the same pattern as in the views and rename this class to `Ids`, let it also inherit from `str, Enum`, remove the comment and the inner class (there is no reason to separate for views here since there are no other IDs), and move it into the `GroupTree` class.

```python
class GroupTree(WebvizPluginABC):
    class Ids(str, Enum):
        GROUP_TREEVIEW = "group-tree-view"
```